doc/org-roam.org

```diff
@@ -370,6 +370,12 @@ time by setting ~org-roam-db-update-idle-seconds~; or change the cache update
 to be triggered immediately after buffer save by setting
 ~org-roam-db-update-method~ to ~'immediate~.
 
+For experienced ~org-capture~ users, the behavior of ~M-x org-roam-find-file~ 
```

Comment:
Just reading this line makes we wonder, `find-file` is a common term in Emacs for finding and visiting file. Maybe the confusing part here is that it uses capture if file doesn't exist 😸

Reply:
Heh, for me, it's was mostly the surprise value of having `find-file` be the capture function described in the docs---there isn't currently any ref to the existence of a capture function that doesn't have find-file behavior, so I sort of assumed that the concept of find-file was being overloaded!  LOL...

Reply:
Haha, indeed 😸 Confused me as well, when I used `org-roam-find-file` for the first time.